pywbem/cim_obj.py

```diff
@@ -3653,33 +3701,43 @@ class CIMQualifierDeclaration(_CIMComparisonMixin):
         This variable will never be `None`.
 
       tosubclass (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value propagates
-        to subclasses.
+        If not `None`, specifies the tosubclass flavor. This flavor defines
+        whether the qualifier value propagates to subclasses. In MOF this
+        flavor is controlled by the `tosubclass` and `restricted` flavor
+        keywords.
 
         `None` means that this information is not available.
 
       toinstance (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value propagates
-        to instances.
+        If not `None`, specifies the toinstance flavor. This flavor defines
+        whether the qualifier value propagates to instances.
 
         `None` means that this information is not available.
 
         Note that :term:`DSP0200` has deprecated the presence of qualifier
         values on CIM instances.
 
       overridable (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value is overridable
-        in subclasses.
+        If not `None`, specifies the  overridable flavor. This flavor defines
+        whether the qualifier value is overridable in subclasses. Internally
+        the flavor is named overridable; setting of this flavor in MOF is defined by
+        the `enableoverride` and `disableoverride` keywords.
```

Comment:
"Overrideable" is not a flavor as defined in DSP0004. We settled on using the MOF keywords also for the flavors in the metamodel, i.e. this attribute specifies the "EnableOverride" and "DisableOverride" flavors. I think we should describe that with explicit text for the values `True`, `False`, and `None`.